app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    user = User.create_invited_user(user_params)
+    if user.save
```

Comment:
Could we roll `user.save` into `User::create_invited_user`? I'm not sure you'd ever want to make an invited user without persisting it. What do you think?